src/components/Display.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div className="display">
+      <input type="text" value={result} />
+    </div>
+  );
+};
+
+Display.defaultProps = {
+  result: 0,
+};
+
+Display.propTypes = {
+  result: PropTypes.number,
```

Comment:
result: PropTypes.number.isRequired

Reply:
I don't think the result is a required prop, as it already has a default prop value. Should I get rid of the default props or make result a required prop?